### PR TITLE
2428 KeyFi Eligibility

### PR DESCRIPTION
**Pull request by andregoncalves**

### CODE REVIEW

File: src/common/marketplaces/categories.js
Comment (sk91) [inline src/common/marketplaces/categories.js:73]: This is bad, thous icons should be inside the source code.

Let's open an issue for it

File: src/renderer/dashboard/index.jsx
Comment (sk91) [inline src/renderer/dashboard/index.jsx:110]: Should this go away once application is approved?

File: src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx
Comment (sk91) [inline src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx:33]: This should go away

**Comment by andregoncalves** [comment]
@sk91 don't merge this yet, I'm making a change to support no payments: https://mattermost.kyc-chain.com/kyc-chain/pl/xnpiudjc7f81j8g1bujupq68tw

**Review by sk91 — APPROVED** [review]
LGTM with some comments

File: src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx
Comment (sk91) [inline src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx:178]: will it work with KI?

File: src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx
Comment (sk91) [inline src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx:34]: Will it work wit KI?

File: src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx
Comment (sk91) [inline src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx:77]: Will it work wit KI?

File: src/common/marketplaces/categories.js
Comment (andregoncalves) [reply src/common/marketplaces/categories.js:73]: Agreed. Issue: https://github.com/SelfKeyFoundation/Identity-Wallet/issues/2440

File: src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx
Comment (andregoncalves) [reply src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx]: Removed

File: src/renderer/dashboard/index.jsx
Comment (andregoncalves) [reply src/renderer/dashboard/index.jsx:110]: Good point.
I pushed a change to hide the widget if an existing application exists (even if it's not approved, because a user can only apply once).

File: src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx
Comment (andregoncalves) [reply src/renderer/marketplace/keyfi/checkout/keyfi-checkout-container.jsx:165]: Some of those props were not needed. It should work with KI.
(Exchange rate is only available using KEY)

File: src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx
Comment (andregoncalves) [reply src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx]: No it wouldn't, replaced it with primaryToken.

File: src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx
Comment (andregoncalves) [reply src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx:78]: Some of those props were not needed. It should work with KI.
(Exchange rate is only available using KEY)

**Review by sk91 — APPROVED** [review]
LGTM

**Pull request merged**
